scene/3d/lightmap_gi.cpp

```diff
@@ -1352,6 +1365,7 @@ void LightmapGI::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_POST_ENTER_TREE: {
 			if (light_data.is_valid()) {
+				ERR_FAIL_COND_MSG(light_data->is_using_spherical_harmonics() && !light_data->is_using_packed_directional(), "The directional lightmap textures are stored in a format that isn't supported anymore. Please rebake to fix the issue.");
```

Comment:
Display resource path and node name to make troubleshooting eassier, and reword the message to describe more clearly what happens in practice:

```suggestion
				ERR_FAIL_COND_MSG(light_data->is_using_spherical_harmonics() && !light_data->is_using_packed_directional(), vformat("%s (%s): The directional lightmap textures are stored in a format that isn't supported anymore. Please bake lightmaps again to make lightmaps display from this node again.", get_light_data()->get_path(), get_name()));
```